src/mococrw/openssl_wrap.h

```diff
@@ -1412,5 +1427,17 @@ std::vector<uint8_t> _i2d_ECSDA_SIG(const ECDSA_SIG*);
  */
 SSL_BIGNUM_Ptr _BN_bin2bn(const uint8_t* data, size_t dataLen);
 
+/**
+ * Write an OpenSSL bignum in big-endian plain bytes representation of an
+ * unsigned integer.
+ *
+ * @param bignum Number to write to
```

Comment:
Rather to serialize? it is not written to the bignum param. In fact, it is const.